pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: collection
-version: 1.17.1
+version: 1.17.2
 description: Collections and utilities functions and classes related to collections.
 repository: https://github.com/dart-lang/collection
 
 environment:
-  sdk: ">=2.18.0 <3.0.0"
+  sdk: ">=2.18.0 <4.0.0"
```

Comment:
This really makes no difference, as we re-write the upper constraint to `4.0.0` anyway for null safe packages.

But don't we want to change the lower constraint too as you are about to start using Dart 3 modifiers?

Reply:
I prefer to not *depend* on constraint re-writing. It's a hack. It's fine transitionally, but I'd like to have a version which opts in to `<4.0.0`.

But I already had a PR for that, so closing this one.